lib/src/components/Logo/Welcome.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+import { classNames } from '@/utils'
+
+import logoStyles from './logo.module.scss'
+import type { LogoProps } from './types'
+
+const cx = classNames(logoStyles)
+
+export const Welcome: React.FC<LogoProps> = props => {
```

Comment:
```suggestion
export const Welcome = (props: LogoProps) => {
```

But do you know why this is the only one without a forwarded ref?

Reply:
No context for this sorry, I harmonized the ref passing logic for every logo